src/hubmap_translation/addl_index_transformations/portal/translate.py

```diff
@@ -23,7 +20,7 @@ def _unexpected(s):
 
 
 def translate(doc):
-    _add_has_visualization(doc)
+    # _add_has_visualization(doc)
```

Comment:
Could you clarify the reason for the removal of the `add_has_visualization` functionality? The ingest API endpoint should be up in all environments.

Reply:
I just forgot to delete the commented out line. The `visualization` bool is added in src/hubmap_translation/addl_index_transformations/portal/add_assay_details.py.